app/src/test/java/com/example/weathertimeapp/modelTest/WeatherDetailsModelTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.weathertimeapp.modelTest
```

Comment:
in these tests you are only checking if the callback finish.. please add validations for the info that must be returned

Reply:
added nullity assertions on the tests.